src/components/OrderCard/OrderCard.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import { Dispatch, SetStateAction } from 'react';
+
+import styles from './OrderCard.module.css';
+
+type OrderCardProps = {
+  table: number;
+  openModal: Dispatch<
```

Comment:
Is this type somehow required? TBH it looks a bit like an overthinking or overengineering for a really easy use case component and function